cdap-app-fabric/src/test/java/io/cdap/cdap/internal/capability/CapabilityManagementServiceTest.java

```diff
@@ -493,7 +506,7 @@ public void testApplicationDeployment() throws Exception {
     capabilityManagementService.runTask();
   }
 
-  @Test
+  //TODO(CDAP-17528): Re-enable this test once fix.
```

Comment:
why do we need to ignore this test now? 
Wondering if there is any underlying issue with implementation or this is just flaky test? If latter, could we fix it?

Reply:
BTW, use `@Ignore` to ignore test

Reply:
https://cdap.atlassian.net/browse/CDAP-17528 is a known issue hence sickbaying the test. The test was not setup correctly it seems.

Reply:
+1 to Terence's comment